site/docs/guide/middleware.md

```diff
@@ -55,9 +55,9 @@ Middleware can be a function or an object.
 We only used functions (`(ctx) => { ... }`) so far, so let's ignore middleware objects for now, and dig deeper into the `MiddlewareFn` type ([reference](https://doc.deno.land/https/deno.land/x/grammy/mod.ts#MiddlewareFn)):
 
 ```ts
-// omitted type parameters again
+// Omitted type parameters again.
 type MiddlewareFn = (ctx: Context, next: NextFunction) => MaybePromise<unknown>;
-// with
+// With
```

Comment:
I think this is an exception, and needs to be kept lowercase and without a period. It will look too weird otherwise.
```suggestion
// with
```